app/assets/javascripts/host_edit.js

```diff
@@ -1,5 +1,12 @@
 //= require parameter_override
 
+var HOST_EDIT_ATTRIBUTES = {
+  "architecture" : ['architecture_id', 'organization_id', 'location_id'],
```

Comment:
I personally don't think a global variable is such a bad idea... redux can be used, but it might be an overkill.

however, i don't really like the use of "var", and prefer to always use "const", to make operations immutable.

the following:
`attrs[type] = (window.tfm.globals.host_edit_attribute_override || DEFAULT_HOST_EDIT_ATTRIBUTES["architecture"]);
`
seems to me much cleaner and safer